.gitignore

```diff
@@ -1,12 +1,104 @@
-# Filetypes
-*~
```

Comment:
Can you re-add this ignore for the "backup" files that Emacs (my editor of choice) creates?

Reply:
I can but then should I add vim and Sublime Text ignore files? I recommend adding the emacs backup files into your [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

Reply:
I'm OK with having multiple editors' worth of crud in the `.gitignore`.